static/utils/contributors.js

```diff
@@ -1,17 +1,88 @@
 const contributors = [
   {
-    name: "John Doe",
-    email: "abc@xyz.com",
-    role: "Lead PEcAn Developer",
-    education: "PhD at MIT, USA",
-    img: "https://previews.123rf.com/images/stockbroker/stockbroker1710/stockbroker171000524/89639450-portrait-of-a-young-smiling-professional-man-arms-crossed.jpg",
+    name: "Dr. Michael Dietze",
```

Comment:
Please suggest over here which people should be added as i have referenced 3 year old data to build this